core/backend/Data/LegacyHandler/PresetDataHandlers/HistoryTimelineDataHandler.php

```diff
@@ -322,6 +322,10 @@ public function fetch(
             $listData[$key]['assigned_user_name']['user_name'] = $user->user_name ?? '';
             $listData[$key]['assigned_user_name']['user_id'] = $record['assigned_user_id'];
             $listData[$key]['module_name'] = $panelToModuleName[$listData[$key]['panel_name']];
+            $listData[$key]['descrition'] = html_entity_decode($listData[$key]['descrition']  ?? '', ENT_QUOTES);
```

Comment:
Is this really `descrition` instead of `description`?

Reply:
Typo has been fixed.